app/scripts/firebase/main.js

```diff
@@ -9,26 +9,58 @@ function ArtHop() {
   this.logoutButton = document.getElementById('logout');
   this.nameDisplay = document.getElementById('sample-text');
   this.venues = [];
+  this.users = [];
 
   //Button listeners
   this.loginButton.addEventListener('click', this.signIn.bind(this));
   this.logoutButton.addEventListener('click', this.signOut.bind(this));
 
   this.initFirebase();
   this.loadVenues();
+  this.loadUsers();
 }
 
 ArtHop.prototype.loadVenues = function() {
   this.venuesRef = this.database.ref('venues');
   this.venuesRef.off();
   var self = this;
-  var venues = [];
 
   this.venuesRef.once('value').then(function(data) {
     self.venues = data.val(); 
   });
 }
 
+ArtHop.prototype.loadUsers = function() {
+  this.usersRef = this.database.ref('users');
+  this.usersRef.off();
+  var self = this;
+
+  function appendUsers(data) {
+    var users = data.val();
+    for(var u in users) {
```

Comment:
I wasn't sure how else to easily iterate over an objects keys. I think this was the simplest way I found but I can look for a cleaner solution.